router/core/graphql_prehandler.go

```diff
@@ -195,6 +209,43 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 	})
 }
 
+func (h *PreHandler) flushMetrics(ctx context.Context, requestLogger *zap.Logger) {
```

Comment:
Is this used in Serverless mode only? or also in regular mode?

Keep in mind that at 10k rps, it's potentially creating 30k goroutines that need to be gc'd.

Reply:
ok